verilog/formatting/tree_unwrapper.cc

```diff
@@ -943,6 +943,14 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
       AttachTrailingSemicolonToPreviousPartition();
       break;
     }
+    case NodeEnum::kTaskHeader: {
+      // Fix kTaskHeader section policy
+      CurrentTokenPartition()->Parent()->ApplyPostOrder(
+        [](verible::VectorTree<UnwrappedLine>& node) {
+          node.Value().SetPartitionPolicy(PartitionPolicyEnum::kAppendFittingSubpartitions);
+        });
+      break;
+    }
```

Comment:
Is it possible to set this policy as partitions are created (earlier in this function), rather than during this fix-up phase?

Reply:
Done. IMHO it's more messy than in during fix-up phase.